src/main/java/lottoGame/controller/LottoStore.java

```diff
@@ -1,60 +1,74 @@
 package lottoGame.controller;
 
 import static lottoGame.view.Casher.askBeforeWinNums;
+import static lottoGame.view.Casher.askBonusNum;
 import static lottoGame.view.Casher.askBuyPrice;
 import static lottoGame.view.Casher.informBuyCount;
 import static lottoGame.view.Casher.informPublishedLottos;
-import static lottoGame.view.Casher.informWinningLottoNums;
+import static lottoGame.view.Casher.informWinResult;
 
+import java.util.List;
+import lottoGame.model.lotto.Lotto;
 import lottoGame.model.lotto.LottoMachine;
+import lottoGame.model.lotto.LottoNum;
 import lottoGame.model.lotto.Lottos;
 import lottoGame.model.price.BuyPrice;
-import lottoGame.model.winner.BeforeWinNums;
 import lottoGame.model.winner.WinnerResult;
-import lottoGame.view.WinResultDto;
 
 public class LottoStore {
-    public static final int PER_LOTTO_PRICE = 1000;
+
+    public static final int PER_LOTTO_PRICE = 1_000;
 
     public void start() {
         BuyPrice buyPrice = getBuyPrice();
+        LottoMachine lottoMachine = new LottoMachine(PER_LOTTO_PRICE);
 
-        Lottos lottos = buyLottos(buyPrice);
+        Lottos lottos = buyLottos(
+                lottoMachine,
+                buyPrice
+        );
 
         totalWinResult(
-                new BeforeWinNums(askBeforeWinNums()),
+                getBeforeWinLotto(lottoMachine),
                 lottos,
-                buyPrice.price()
+                buyPrice.price(),
+                new LottoNum(askBonusNum())
         );
     }
 
-    private Lottos buyLottos(BuyPrice buyPrice) {
-        Lottos lottos = publishLottos(buyPrice);
+    private Lottos buyLottos(LottoMachine lottoMachine, BuyPrice buyPrice) {
+        Lottos lottos = lottoMachine.publish(buyPrice);
         informPublishedLottos(lottos.convertRawString());
         return lottos;
     }
 
+
     private BuyPrice getBuyPrice() {
         BuyPrice buyPrice = new BuyPrice(askBuyPrice());
         informBuyCount(buyPrice.calculateLottoCount(PER_LOTTO_PRICE));
 
         return buyPrice;
     }
 
-    private Lottos publishLottos(BuyPrice buyPrice) {
-        LottoMachine lottoMachine = new LottoMachine(PER_LOTTO_PRICE);
+    private Lotto getBeforeWinLotto(LottoMachine lottoMachine) {
+        List<LottoNum> lottoByNums = lottoMachine.createLottoByNums(askBeforeWinNums());
 
-        return lottoMachine.publish(buyPrice);
+        return new Lotto(lottoByNums);
     }
 
-    private void totalWinResult(BeforeWinNums beforeWinNums, Lottos lottos, int price) {
-        WinnerResult winnerResult = lottos.compareAndElectWinResult(beforeWinNums);
+    private void totalWinResult(
+            Lotto beforeWinLotto,
+            Lottos lottos,
+            int price,
+            LottoNum bonusNum
```

Comment:
```suggestion
            WinnerResult winnerResult,
            int price
```
메서드 인자가 많은데 위와 같이 결과를 전달하도록 접근해도 되지 않을까?